### PR TITLE
[Azure Maps] - Onboarding for Route SDKs

### PR DESCRIPTION
**Pull request by alextts627**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/maps/azure-maps-route/README.md [hunk 1]
```diff
@@ -78,61 +80,140 @@ route_client = MapsRouteClient(credential=credential)
...
+objects are async context managers and define async `close` methods.
+
 ## Examples
 
 The following sections provide several code snippets covering some of the most common Azure Maps Route tasks, including:
 
-- [Get Route Directions](#get-route-directions)
-- [More Sample Code](#sample-code)
+- [Request and Get Route Directions](#request-and-get-route-directions)
+
```
Comment (dubiety) [inline sdk/maps/azure-maps-route/README.md:103]: Suggest to remove empty line between list of items (line 103 and line 105)

Comment (alextts627) [reply sdk/maps/azure-maps-route/README.md]: removed.

**Comment by alextts627** [comment]
> **API change check**
> 
> APIView has identified API level changes in this PR and created following API reviews.
> 
> [azure-maps-route](https://apiview.dev/Assemblies/Review/d11ba093ee9744fa935fb129351cde99)

Previous [APIview](https://apiview.dev/Assemblies/Review/e6fab45424f74dc89b6064001b034713) for the board review.

**Comment by alextts627** [comment]
Previous [APIview](https://apiview.dev/Assemblies/Review/e6fab45424f74dc89b6064001b034713) for the board review.

File: sdk/maps/azure-maps-route/README.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+Async clients and credentials should be closed when they're no longer needed. These
+objects are async context managers and define async `close` methods.
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Azure Maps Route tasks, including:
+
+- [Request and Get Route Directions](#request-and-get-route-directions)
+- [Reqest and Get Route Range](#reqest-and-get-route-range)
```
Comment (dubiety) [inline sdk/maps/azure-maps-route/README.md:103]: Typo

File: sdk/maps/azure-maps-route/README.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+This service request returns a route between an origin and a destination, passing through waypoints if they are specified. The route will take into account factors such as current traffic and the typical road speeds on the requested day of the week and time of day.
+
+```python
+from azure.maps.route import MapsRouteClient
+
+route_directions_result = client.get_route_directions(route_points=[LatLon(47.60323, -122.33028), LatLon(53.2, -106)]);
+```
+
+### Reqest and Get Route Range
```
Comment (dubiety) [inline sdk/maps/azure-maps-route/README.md:117]: Typo

File: sdk/maps/azure-maps-route/README.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+### More sample code
+
+Get started with our [Maps Route samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-route/samples) ([Async Version samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-route/samples/async_samples)).
+
+Several Azure Maps Route Python SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Maps Route
+
+```bash
+set AZURE_SUBSCRIPTION_KEY="<RealSubscriptionKey>"
+
+pip install azure-maps-route --pre
```
Comment (dubiety) [inline sdk/maps/azure-maps-route/README.md:203]: We have `--pre` in Route PR but not in Search PR. Do you think we can sync the description in these two PRs?

Comment (alextts627) [reply sdk/maps/azure-maps-route/README.md:203]: Aligned with in Search SDK.

**Review by dubiety — APPROVED** [review]

File: sdk/maps/azure-maps-route/setup.py [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+#--------------------------------------------------------------------------
+
+import re
+import os.path
+from io import open
+from setuptools import find_packages, setup
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-maps-route"
+PACKAGE_PPRINT_NAME = "Azure Maps Route"
```
Comment (dubiety) [inline sdk/maps/azure-maps-route/setup.py:16]: Duplicate "Azure" in line 16 and line 56, should remove "Azure" from line 16 according to azure-template

Comment (dubiety) [reply sdk/maps/azure-maps-route/setup.py:16]: Same for search and render package

File: sdk/maps/azure-maps-route/dev_requirements.txt [hunk 6]
```diff
@@ -0,0 +1,5 @@
+-e ../../../tools/azure-devtools
+-e ../../../tools/azure-sdk-tools
+-e ../../core/azure-core
+-e ../../identity/azure-identity
+aiohttp>=3.0; python_version >= '3.5'
```
Comment (dubiety) [inline sdk/maps/azure-maps-route/dev_requirements.txt:5]: Curious why python_version not >= 3.6?

**Review by tjprescott — CHANGES_REQUESTED** [review]
Holding this PR until I have the opportunity to review.

**Review by tjprescott — CHANGES_REQUESTED** [review]
Holding this PR until I have the opportunity to review.

**Comment by tjprescott** [comment]
Assuming this is the latest APIView:
https://apiview.dev/Assemblies/Review/c46c5bcae8a3467daa51668b6ba8c820

@praveenkuttappan why is it generating two?

**Review by tjprescott — CHANGES_REQUESTED** [review]
From the APIView it looks like this was generated using classic Autorest V2 whereas Render was generated using Python DPG. Is there a reason this is not?

**Review by tjprescott — APPROVED** [review]
LGMT for beta release. Will need a closer look for GA.

**Pull request merged**